user/serializers.py

```diff
@@ -22,3 +25,30 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField()
+    password = serializers.CharField(
+        style={"input_type": "password"}, trim_whitespace=False
+    )
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(
+                request=self.context.get("request"),
+                username=email,
```

Comment:
In the `AuthTokenSerializer`, the `authenticate` function uses `username=email`. While this is technically correct, it can be confusing. Consider adding a comment to clarify that `username` is used for email-based authentication.